src/main/java/xyz/nucleoid/plasmid/game/StartResult.java

```diff
@@ -6,38 +6,64 @@
 
 import javax.annotation.Nullable;
 
+/**
+ * Describes the result of a {@link GameWorld} attempting to start.
+ */
 public final class StartResult {
+    public static final StartResult ALREADY_STARTED = error(new LiteralText("This game has already started!"));
+    public static final StartResult NOT_ENOUGH_PLAYERS = error(new LiteralText("Game does not have enough players yet!"));
+
     private final Text error;
 
     private StartResult(Text error) {
         this.error = error;
     }
 
+    /**
+     * Returns a {@link StartResult} with no error.
+     *
+     * @return {@link StartResult} with no error (null)
```

Comment:
I think the implementation of using a null error should be hidden- this just represents a successful result